src/main/resources/db/migration/V0007__Insert_test_data_animal_images.sql

```diff
@@ -0,0 +1,25 @@
+INSERT INTO `animal_images` VALUES (1,3,'testPath','jpeg',100500,'2023-08-23 08:39:35','2023-08-23 08:39:35'),
+(6,3,'testPath999','jpeg',100500,'2023-08-23 08:41:38','2023-08-23 08:41:38'),
+(7,1,'test2','png',100599,'2023-08-23 08:41:38','2023-08-23 08:41:38'),
+(8,3,'testPath999','jpeg',100500,'2023-08-23 08:43:04','2023-08-23 08:43:04'),
+(9,1,'test2','png',100599,'2023-08-23 08:43:04','2023-08-23 08:43:04'),
+(13,1,'W:\\Java study\\Java Super Pro\\uploadImages\\Sertificate Hillel Basic.png','png',310346,'2023-08-28 12:40:53','2023-08-28 12:40:53'),
+(14,1,'W:\\Java study\\Java Super Pro\\uploadImages\\Sertificate Hillel Basic.png','png',310346,'2023-08-28 13:15:31','2023-08-28 13:15:31'),
```

Comment:
Why you use absolute path to folder ?

Reply:
This is imitation of outside storage path. But maybe save inside project... I think, that better outside storage

Reply:
done